en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -8,88 +8,94 @@ Performing root cause analysis is crucial in identifying and rectifying the unde
 ## Get started with the sample service
 
 1. Go to **Services** from the left navigation.
-2. If this is your first service, then enter a name for the service and click **Create**. Skip step 3 and go to step 4.
-       ![Create a new service](../assets/img/observability/first-service-creation.png)
-3. Else, click **+ Create** to create a new service.
-       ![Create a new service](../assets/img/observability/service-creation.png)
-    1. Enter a name for the service and click **Create**.
-4. From the left-hand menu, click **Observe**.
-5. On the right-hand bottom, click the **Sample Service** icon.
+2. Create a new service.
+    1. **If this is your first service**, 
+           1. Enter a name for the service and click **Create**. 
+           ![Create a new service](../assets/img/observability/first-service-creation.png)                        
+    2. **Else** (if you already have services created), 
+           1. Click **+ Create** to start creating a new service.
+           2. Enter a name for the service and click **Create**.
+           ![Create a new service](../assets/img/observability/service-creation.png)
+
+3. From the left-hand menu, click **Observe**.
+4. On the right-hand bottom, click the **Sample Service** icon.
        ![Sample service popup](../assets/img/observability/sample-service-pop-up.png)
-6. Click **Try Sample Service**.
+5. Click **Try Sample Service**.
 
        ![Try sample service](../assets/img/observability/try-sample-service.png)
     
-## Detecting anomalies and performing root cause analysis
-You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latency of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
+## Detect anomalies and perform root cause analysis
+You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latencies of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
 
-### Analyzing the throughput graph
-The throughput graph will show the throughput of requests per hour for a selected timestamp. 
+### Step 1: Detect the anomalies
 
-With the sample data, you will observe spikes in the throughput graph. Let's see how you can find
-the root cause by analyzing this data.
+#### Analyze the throughput graph
+The throughput graph will show the throughputs of requests per hour for a selected timestamp. 
```

Comment:
Why this difference? 
<img width="871" alt="image" src="https://user-images.githubusercontent.com/16300038/118082625-20892280-b3db-11eb-8eb0-216f61072d29.png">

Reply:
Fixed